tests/test_eval.py

```diff
@@ -120,7 +120,7 @@ def test_compute_confidence_prediction_distribution(evaluator_mobilenet):
     actual = str(exception).split('ValueError: ')[1]
     assert actual == expected
 
-    probabilities, labels = evaluator_mobilenet.evaluate(os.path.abspath('tests/files/catdog/test'))
+    _, _ = evaluator_mobilenet.evaluate(os.path.abspath('tests/files/catdog/test'))
```

Comment:
If you don't care about the output you don't need to do any assignment. Just write:
```
evaluator_mobilenet.evaluate(os.path.abspath('tests/files/catdog/test'))
```